akka-http-core/src/test/scala/akka/http/impl/engine/client/HttpConfigurationSpec.scala

```diff
@@ -154,8 +154,8 @@ class HttpConfigurationSpec extends AkkaSpec {
           akka.http.host-connection-pool.max-connections = 1
         """.stripMargin) { sys ⇒
 
-          intercept[IllegalArgumentException] { ConnectionPoolSettings(sys) }
-        }
+        intercept[IllegalArgumentException] { ConnectionPoolSettings(sys) }
+      }
     }
   }
 
```

Comment:
meh, will remove files from the PR that do not contribute to the feature tomorrow